src/Sql/Auth/dbo/Stored Procedures/Grant_Save.sql

```diff
@@ -1,5 +1,5 @@
 ﻿CREATE PROCEDURE [dbo].[Grant_Save]
-    @Key  NVARCHAR(200),
+  @Key  NVARCHAR(200),
```

Comment:
⛏️ Go ahead and open this file in an editor with Prettier support. I prefer VS Code. It'll keep this straight.

Separately, I feel our SQL formatting isn't really defined and we could work on that e.g. why are two spaces after some of these parameter names allowed?